paderbox/array/interval/core.py

```diff
@@ -701,6 +764,9 @@ def sum(self, axis=None, out=None):
             sum = self.shape[0] - sum
         return sum
 
+    def mean(self, axis=None, out=None):
+        return self.sum(axis, out) / self.shape[0]
```

Comment:
This fails when someone adds support for `out` to `self.sum`. I guess it's fine here, but we should keep that in mind.

Reply:
Fixed